src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/service/zookeeper/ZooKeeperService.java

```diff
@@ -44,7 +44,7 @@ public class ZooKeeperService implements IService, LifeCycleObserver {
     // management expected state consists of:
     // 1 producer
     // 5 consumers for following topics: kilda.speaker, kilda.speaker.flow, kilda.speaker.flow.ping,
-    // kilda.speaker.disco, kilda.speaker.switch.manager
+    // kilda.network.control, kilda.speaker.switch.manager, kilda.speaker.network
```

Comment:
`kilda.network.control`, but `kilda.speaker.network` ... these names use different build patterns. Is it intended?

Reply:
No, fixed.